.github/ISSUE_TEMPLATE/introduce-sgid-dataset.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Introduce data to the SGID
+about: Creation Checklist
+title: Add <dataset name> from <place>
+labels: 'introduction'
+assignees: ''
+
+---
+# Summary
+
+A short summary of the situation...
+
+<!-- 
+1. The three teams will triage the request, striking through any line that does not apply.
+1. Assign each step to someone based on list of assignments
+1. All checkboxes should either be striked out or checked before closing the issue.
+-->
+
+# Triage
+
+- [ ] Data Team Triage
+- [ ] Dev Team Triage (assigned to)
+- [ ] Cadastre Team Triage
+
+# Data is live in the following areas
+
+- [ ] ArcGIS Online (assigned to)
```

Comment:
Automatically done via AGOL Pallet.
```suggestion
```

Reply:
I don't think this should be removed. I think it should be reorganized. So I am going to reject this suggestion.